cloud/deployment/deployment.go

```diff
@@ -91,102 +105,119 @@ func CanCiCdDeploy(bearerToken string) bool {
-			tab.AddRow([]string{d.Label, releaseName, clusterName, d.ID, runtimeVersionText, strconv.FormatBool(d.DagDeployEnabled), strconv.FormatBool(d.APIKeyOnlyDeployments)}, false)
+			tab.AddRow([]string{d.Name, releaseName, clusterName, cloudProvider, region, d.Id, runtimeVersionText, strconv.FormatBool(d.DagDeployEnabled), strconv.FormatBool(d.IsCicdEnforced), string(*d.Type)}, false)
 		}
 	}
-
-	return tab.Print(out)
+	tab.Print(out)
+	return nil
 }
 
-func Logs(deploymentID, ws, deploymentName string, warnLogs, errorLogs, infoLogs bool, logCount int, client astro.Client) error {
-	logLevels := []string{}
-
+func Logs(deploymentID, ws, deploymentName string, warnLogs, errorLogs, infoLogs bool, logCount int, platformCoreClient astroplatformcore.CoreClient, coreClient astrocore.CoreClient) error {
+	var logLevel string
+	var i int
 	// log level
 	if warnLogs {
-		logLevels = append(logLevels, "WARN")
+		logLevel = "WARN"
+		i++
 	}
 	if errorLogs {
-		logLevels = append(logLevels, "ERROR")
+		logLevel = "ERROR"
+		i++
 	}
 	if infoLogs {
-		logLevels = append(logLevels, "INFO")
+		logLevel = "INFO"
+		i++
 	}
-	if len(logLevels) == 0 {
-		logLevels = []string{"WARN", "ERROR", "INFO"}
+	if i > 1 {
+		return errors.New("cannot query for more than one log level at a time")
 	}
 
 	// get deployment
-	deployment, err := GetDeployment(ws, deploymentID, deploymentName, false, client, nil)
+	deployment, err := GetDeployment(ws, deploymentID, deploymentName, false, platformCoreClient, nil)
 	if err != nil {
 		return err
 	}
 
-	deploymentID = deployment.ID
-
-	// deployment logs request
-	vars := map[string]interface{}{
-		"deploymentId":  deploymentID,
-		"logCountLimit": logCount,
-		"start":         "-24hrs",
-		"logLevels":     logLevels,
+	deploymentID = deployment.Id
+	timeRange := 86400
+	offset := 0
+	getDeploymentLogsParams := astrocore.GetDeploymentLogsParams{
+		Sources: []astrocore.GetDeploymentLogsParamsSources{
+			"scheduler",
+			"webserver",
+			"triggerer",
+			"worker",
+		},
+		MaxNumResults: &logCount,
+		Range:         &timeRange,
+		Offset:        &offset,
+	}
+	if logLevel != "" {
+		getDeploymentLogsParams.SearchText = &logLevel
 	}
 
-	deploymentHistoryResp, err := client.GetDeploymentHistory(vars)
+	deploymentLogs, err := GetDeploymentLogs("", deploymentID, getDeploymentLogsParams, coreClient)
 	if err != nil {
-		return errors.Wrap(err, astro.AstronomerConnectionErrMsg)
+		return err
 	}
 
-	Logs := deploymentHistoryResp.SchedulerLogs
-
-	if len(Logs) == 0 {
-		fmt.Println("No matching logs have been recorded in the past 24 hours for Deployment " + deployment.Label)
+	if len(deploymentLogs.Results) == 0 {
+		fmt.Println("No matching logs have been recorded in the past 24 hours for Deployment " + deployment.Name)
 		return nil
 	}
-
-	fmt.Println(Logs)
-
+	for i := range deploymentLogs.Results {
+		fmt.Printf("%f %s %s\n", deploymentLogs.Results[i].Timestamp, deploymentLogs.Results[i].Raw, deploymentLogs.Results[i].Source)
```

Comment:
so it will look something like:
`January 1, 2024 13:15:00 [WARN] this is a log scheduler` ?

Reply:
It looks like this for example:
<img width="1338" alt="Screenshot 2024-01-11 at 10 39 24 AM" src="https://github.com/astronomer/astro-cli/assets/63181127/9ab0ed25-f4c5-4dff-bf43-ad9e6679670a">